src/lambda.js

```diff
@@ -9,8 +9,17 @@ const { promisify } = require('util');
 
 const LAMBDA_IMAGE = 'lambci/lambda:nodejs6.10';
 
-const createEnvironmentVariables = (environment) => Object.entries(environment)
-  .map(([ key, value ]) => `${key}=${value}`);
+const createEnvironmentVariables = (environment) => {
+  const envVariables = Object.entries(environment)
+    .map(([ key, value ]) => `${key}=${value}`);
+  if (environment.DISABLE_XRAY_LOGGING) {
+    // We want to disable xray logging in integration tests, because it's too verbose, see
+    // xray set up in lambda-runtime-tools
+    // From Docker docs: a variable without = is removed from the environment, rather than to have an empty value.
+    envVariables.push('AWS_XRAY_CONTEXT_MISSING');
```

Comment:
Have you tested this locally? My understanding is that the default behavior is to throw an exception. See https://github.com/aws/aws-xray-sdk-node/blob/6f310929a42a0c538c21517552d3d2aa18d14a9e/packages/core/lib/context_utils.js#L203

Reply:
yeah, I have tested this locally. The default behavior is to throw an exception, but it doesn't throw an exception in integration tests since we set the setContextMissingStrategy to "()=>{}" in lambda-runtime-tools

So, if someone only requires this project, but not the lambda-runtime-tools, and sets DISABLE_XRAY_LOGGING to true, then he will get exceptions. So, the variable name makes sense when looking at those 2 projects, but it doesn't make sense when just looking at lambda-tools. Do you think it would be better to have 2 separate variables for that reason? 1 would be "UNSET_AWS_XRAY_CONTEXT_MISSING", another one would be "DISABLE_XRAY_LOGGING"

Reply:
I've pushed a new commit where I use unsetXrayContextMissing option instead of an environment variable DISABLE_XRAY_LOGGING to avoid confusion

Reply:
Jeff expressed his concern that lambda-tools is an open source project and because of that it might not make sense to do this change as it is done currently. Copying my suggestion from Slack here:

"Zhenya Ragulin [12:15 PM]
I guess the most flexible way to do it would be to have a parameter “envVariablesToUnset” in the options argument to “useNewContainer” with an array of variable names. This way it will allow us to achieve what we want, and it will keep the lambda-tools project generic enough (edited)

let me know if that makes sense, and if it does, I will change the implementation to do that instead

Zhenya Ragulin [1:28 PM]
another approach is to modify lambda-tools to treat env variables with null or undefined values as those that need to be deleted"

Reply:
Got feedback from Jeff - he likes the null/undefined approach since it's consistent with how docker handles things, going to update my PR